src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50;
+  height: 60;
+}
+
+nav {
```

Comment:
better to use classes instead of tag name

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3647/files#r1280741759
didn't fix it